make-everyvoice-env

```diff
@@ -158,15 +158,15 @@ r pip install cython
 
 # pycountry and pyworld don't always compile cleanly, but we can fall back to
 # using conda-forge if necessary
-PY_COUNTRY_WORLD=$(grep "pycountry\|pyworld" requirements.txt)
+PY_COUNTRY_WORLD=$(grep -o "\(pycountry\|pyworld\)==[0-9.]\+" pyproject.toml)
 if ! r pip install $PY_COUNTRY_WORLD; then
     echo Falling back to installing pycountry and pyworld from conda-forge
     r conda install -y $PY_COUNTRY_WORLD -c conda-forge
 fi
 
 r conda install -y sox -c conda-forge
 r pip install -e .
-r pip install -r requirements.dev.txt
+r pip install .[dev]
```

Comment:
Might as well turn this into one `-e .[dev]` line.

Reply:
In fact, doesn't installing `.[dev]` replace the editable install by a non-editable one?